Community/Service Requests/manage_service_requests/__init__.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2018 BlueCat Networks. All rights reserved.
```

Comment:
Please change date to 2019. Also in manage_service_requests_page.css, manage_service_requests_page.js, manage_service_requests_form.py, manage_service_requests_page.html, __init__.py, service_request_host_page.css, service_request_host_page.py, and service_request_host_page.html.

Reply:
Whoops, missed the 2018 ones! Should be covered on 2018 and 2017 now.